src/main/java/com/esri/samples/scene/choose_camera_controller/ChooseCameraControllerSample.java

```diff
@@ -0,0 +1,192 @@
+
+import java.io.File;
+
+public class ChooseCameraControllerSample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage){
+
+    try {
+
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Choose Camera Controller Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createImagery());
+
+      // set the scene to a scene view
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // add base surface for elevation data
+      Surface surface = new Surface();
+      ArcGISTiledElevationSource elevationSource = new ArcGISTiledElevationSource("http://elevation3d.arcgis" +
+              ".com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer");
+      surface.getElevationSources().add(elevationSource);
+      scene.setBaseSurface(surface);
+
+      // create a graphics overlay for the scene
+      GraphicsOverlay sceneGraphicsOverlay = new GraphicsOverlay();
+      sceneView.getGraphicsOverlays().add(sceneGraphicsOverlay);
+
+      // create a graphic with a ModelSceneSymbol of an aeroplane to add to the scene
+      String modelURI = new File("./samples-data/bristol/Collada/Bristol.dae").getAbsolutePath();
+      ModelSceneSymbol plane3DSymbol = new ModelSceneSymbol(modelURI, 1.0);
+      plane3DSymbol.loadAsync();
+      plane3DSymbol.setHeading(45);
+      Graphic plane3D = new Graphic(new Point(-109.937516, 38.456714, 5000, SpatialReferences.getWgs84()), plane3DSymbol);
+      sceneGraphicsOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.ABSOLUTE);
+      sceneGraphicsOverlay.getGraphics().add(plane3D);
+
+      // create a camera and set it as the viewpoint for when the scene loads
+      Camera camera = new Camera(38.459291, -109.937576, 5500, 150.0, 20.0, 0.0);
+      sceneView.setViewpointCamera(camera);
+
+      // instantiate a new camera controller which orbits the aeroplane at a set distance
+      OrbitGeoElementCameraController orbitAeroplaneCameraController = new OrbitGeoElementCameraController(plane3D, 100.0);
+      orbitAeroplaneCameraController.setCameraPitchOffset(30);
+      orbitAeroplaneCameraController.setCameraHeadingOffset(150);
+
+      // instantiate a new camera controller which orbits a target location
+      Point locationPoint = new Point(-109.929589, 38.437304, 3500, SpatialReferences.getWgs84());
+      OrbitLocationCameraController orbitLocationCameraController = new OrbitLocationCameraController(locationPoint, 10);
```

Comment:
This is too close. The location should be on the ground and the camera should be much farther away if the idea is to orbit around that basin.